### PR TITLE
Update owners files

### PR DESCRIPTION
**Pull request by cblecker**
Swap in new FLs/TLs. Manually add @jewzaam to approvers so that he can still approve changes.

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -2,14 +2,15 @@
 
 aliases:
   srep-functional-leads:
-    - cblecker
     - dofinn
+    - dustman9000
+    - fahlmant
     - jaybeeunix
-    - lisa
+    - jharrington22
```
Comment (fahlmant) [inline OWNERS_ALIASES:9]: I don't think James is needed here since he's below on level 15

Comment (cblecker) [reply OWNERS_ALIASES:9]: @fahlmant He's listed twice because he's technically the FL of Omega too

Comment (fahlmant) [reply OWNERS_ALIASES:9]: Ah got it :+1:

**Review by fahlmant — APPROVED** [review]
/lgtm

**Pull request merged**